owncloudApp/src/main/java/com/owncloud/android/presentation/sharing/shares/PublicShareDialogFragment.kt

```diff
@@ -871,13 +901,30 @@ class PublicShareDialogFragment : DialogFragment() {
         if (!hasPasswordEnforcedFor && capabilities?.filesSharingPublicPasswordEnforced == CapabilityBooleanType.TRUE) {
             setPasswordEnforced()
         }
+
+        capabilities?.passwordPolicy?.let { passwordPolicy ->
```

Comment:
The `setOnClickListener` could be reformatted.

Reply:
Not that important (KtLint won't complain about it), but ok